e2e/bitrise.yml

```diff
@@ -50,6 +49,31 @@ workflows:
     before_run:
     - _setup
     steps:
+    - path::./:
+        title: Step Test - Multi Target API Key auto signing
+        inputs:
+        - distribution_method: development
+        - archive_path: ./archives/Fruta.xcarchive
+        - product: app
+        - automatic_code_signing: api-key
+        - min_profile_validity: 30
+        - certificate_url_list: $BITFALL_APPLE_IOS_CERTIFICATE_URL_LIST
+        - passphrase_list: $BITFALL_APPLE_IOS_CERTIFICATE_PASSPHRASE_LIST
+    - path::./:
+        title: Step Test - TV OS Apple ID auto signing
+        inputs:
+        - distribution_method: development
+        - archive_path: ./archives/TVOS.xcarchive
+        - min_profile_validity: 30
+        - automatic_code_signing: apple-id
+        - certificate_url_list: $BITFALL_APPLE_IOS_CERTIFICATE_URL_LIST
+        - passphrase_list: $BITFALL_APPLE_IOS_CERTIFICATE_PASSPHRASE_LIST
+    - certificate-and-profile-installer:
+        inputs:
+        - certificate_url: $BITFALL_APPLE_IOS_CERTIFICATE_URL_LIST
+        - certificate_passphrase: $BITFALL_APPLE_IOS_CERTIFICATE_PASSPHRASE_LIST
+        - provisioning_profile_url: $BITFALL_APPLE_PROVISIONING_PROFILE_URL_LIST
+        - install_defaults: "no"
```

Comment:
I was wondering if it worth splitting up the single `test_ipa_exports` workflow into multiple scenarios? 

I am only asking because as I was looking at this I thought that if we make an error in the first step where we use automatic code signing with the Fruta app we could potentially download more signing assets in an error which will be picked up for example by the second tvOS archive signing. So I was thinking if it is worth splitting them and maybe do some cleanup between the workflow runs.

Like we could have various scenarios:
1.) `certificate-and-profile-installer` is before the automatic code signing to test that code signing still works if the user manually installs the certs but still sets up automatic code signing

2.) Fruta is first exported with automatic code signing and then only the Fruta App Clip is exported without automatic code signing to check if the local signing assets are picked up

And probably there are more but did not want to go into great details and just wanted show what I was thinking. 

Introducing more workflows will significantly slow down the e2e tests so this is really just a discussion about it.